convert/tests/test_vertex/test_data/badgerdoc_annotation.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
These tests with pytest fail, when I run them in docker-compose container.
Please re-check it and make sure it works fine in docker-compose container:

FAILED tests/test_vertex/test_import.py::test_render_pdf - AssertionError: assert b'%PDF-1.7\n%...bj\n\nxref\n0' == b'%PDF-1.7\n%...001 f \n00000'
FAILED tests/test_vertex/test_import.py::test_annotation_converter - assert False

Reply:
Fixed